flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -71,6 +71,10 @@
 #define RCVR_ACTIVITY_MONITOR_CHANNELS_PER_GROUP 12
 #define RCVR_ACTIVITY_MONITOR_MIN_RANGE 20
 
+// If the neutral point on the throttle channel is 35% over the configured
+// minimum, we assume they're providing bidirectional thrust.
+#define THRUST_BIDIR_THRESH 0.35f
```

Comment:
We infer what the user wants re: 3D thrust.  If throttle neutral is low, use the old unipolar behavior.  If throttle neutral is in the middle, they want bipolar.